packages/vant/src/slider/test/index.spec.ts

```diff
@@ -374,3 +374,28 @@ test('should update modelValue correctly after clicking the reversed vertical sl
   trigger(wrapper, 'click', 0, 100);
   expect(wrapper.emitted('update:modelValue')!.pop()).toEqual([0]);
 });
+
+//https://github.com/youzan/vant/issues/13625
+test('should return max when distanceToMax <= distanceToPrev', () => {
+  const wrapper = mount(Slider, {
+    props: { min: 0, max: 50, step: 60, modelValue: 45 },
+  });
+
+  const emitted = wrapper.emitted('update:modelValue');
+  if (emitted && emitted.length > 0) {
+    const result = emitted[emitted.length - 1][0] as number;
+    expect(result).toBe(50); // Should return max
+  }
+});
+
+test('should return prevSteppedValue when distanceToMax > distanceToPrev', () => {
+  const wrapper = mount(Slider, {
+    props: { min: 0, max: 8, step: 12, modelValue: 2 },
+  });
+
+  const emitted = wrapper.emitted('update:modelValue');
+  if (emitted && emitted.length > 0) {
+    const result = emitted[emitted.length - 1][0] as number;
+    expect(result).toBe(0); // Should return prevSteppedValue
+  }
+});
```

Comment:
The newly added second test case did not enter the `if(steppedValue > max)` conditional branch. When I commented out the Slider-related modifications and ran the new test case, the test still passed.